snippets/curl/get_nb_assets.sh

```diff
@@ -0,0 +1,116 @@
+!/bin/sh
+
+#####################n#####################################################
```

Comment:
Is the _n_ significant?

```suggestion
###########################################################################
```